From e5d4d187ec7c2bfe344ccb89850683ab00d424d2 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Mon, 20 Jul 2020 16:32:12 +0800
Subject: [PATCH 1/1] Test case for library .asm file

Description: Change the address value in the ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm

Repo: edk2

Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm b/ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm
index 1265dddea636..e9e7d2a169a1 100644
--- a/ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm
+++ b/ArmPkg/Library/ArmLib/Arm/ArmLibSupport.asm
@@ -73,7 +73,7 @@
   bx      lr
 
  RVCT_ASM_EXPORT ArmSetTTBR0
-  mcr     p15,0,r0,c2,c0,0
+  mcr     p15,0,r0,c2,c0,1
   isb
   bx      lr
 
-- 
2.27.0.windows.1

